### DISCUSSION TITLE
Discord callback causes 500

### DISCUSSION THREAD

**Discussion by mbroekman** [requarks/wiki#3175] [Help / Questions]
Hi, i'm trying to activate the auth with discord but get a 500 error as soon as the calback is done by discord. It's furstrating that the only message you get is 'Oops, something went wrong...'

I'm new to docker and Wiki.js and can't find any logging done by Wiki.  Is there any way to find more loggin within Wiki or someone who can point me in the right direction to solve this ?

OS: Ubuntu 20.04.2 LTS and Docker
Wiki.js version: 2.5.170 ( docker requarks/wiki:2)
Database engine: PostgreSQL 11.11

Regards,
Machiel

![image](https://user-images.githubusercontent.com/4540115/109980048-ce06f800-7cff-11eb-9bd3-b19c6e008854.png)

**Comment by TheBrainOne** [comment]
I have same problem.
Host return 500 status code.
